src/main/proto/ga4gh/vrsatile/v1/vrsatile.proto

```diff
@@ -67,21 +55,15 @@ enum MoleculeContext {
 
 // https://vrsatile.readthedocs.io/en/latest/value_object_descriptor/vod_index.html#gene-descriptor
 message GeneDescriptor {
-//  Gene gene = 1;
-  ValueObjectDescriptor valueObjectDescriptor = 1;
+  // The official gene identifier as designated by the organism gene nomenclature committee e.g. HGNC:3477 or MGI:2385071
+  // This should be a CURIE linking the reference to a namespace where it can be retrieved.
+  // Takes place of value_id field in a generic descriptor
+  string gene_id = 1;
```

Comment:
Rename `id`? This mirrors the `VariationDescriptor.id` above and matches the original Phenopacket `Gene.id`

Reply:
For me, alternatide_id means "the same thing" and xref means "a related thing". I am not sure if others see things this way, but it would be good to clarify what we mean here.

Reply:
Good point. This is described in the [docs](https://vrsatile.readthedocs.io/en/latest/value_object_descriptor/vod_index.html#value-object-descriptor) as a 

> List of CURIEs representing associated concepts

So not really specified.

Reply:
Re: comment from @julesjacobsen:
> Rename `gene_id` as `id`? This mirrors the `VariationDescriptor.id` above and matches the original Phenopacket `Gene.id`

A descriptor `id` and `value_id` fields have subtle but distinctly different meanings. The nice thing about it is that you can have any identifiable gene concept, e.g. [CIViC gene 5 (BRAF)](https://civicdb.org/events/genes/5/summary#gene) as the `id`, and then the associated authoritative gene identifier (hgnc:1097) as the `value_id`. It is not applicable to all sources, and Phenopackets may not care for that distinction; however, we should strive to preserve field names across applications of VRSATILE.

I think this should be changed to `value_id` for consistency.

Reply:
Re: comment from @pnrobinson 
> For me, alternatide_id means "the same thing" and xref means "a related thing". I am not sure if others see things this way, but it would be good to clarify what we mean here.

I think this is a good change. Will do. 👍